sunpy/map/__init__.py

```diff
@@ -1,12 +1,13 @@
-"""SunPy Maps"""
-import sys
+"""
+SunPy Map
 
-from sunpy.map.mapbase import GenericMap  # isort:skip
+isort:skip_file
```

Comment:
This looks like a mistake? It seems to have sorted something that was supposed to be skipped?

Reply:
It hasn't sorted it I just changed it to skip the whole file so the diff got weird.